mitodl/mitxpro
# #258 Home Page: Inquire Now

#### Pre-Flight checklist


#### What are the relevant tickets?
#258 

#### What's this PR do?
This PR adds a "dark theme" to the `Text-Image Section` subpage that is used as the "Inquire Now" section on the home page and the "For Teams" section on the course page. Default is the light theme (light background, dark text) that was developed for the course page. This PR adds the ability to switch to dark theme (dark background, light theme).

#### How should this be manually tested?
Add a `Text-Image Section` type subpage to the HomePage. There is a "Dark Theme" checkbox in the CMS which can be selected to switch themes. Selecting the checkbox should display the section with the appropriate appearance.

#### Screenshots (if appropriate)
![Screenshot from 2019-05-23 11-41-27](https://user-images.githubusercontent.com/45350418/58231292-141fe100-7d50-11e9-89b3-6dcae61f54fa.png)
![Screenshot from 2019-05-23 11-40-02](https://user-images.githubusercontent.com/45350418/58231300-171ad180-7d50-11e9-9273-33361514e75a.png)
![Screenshot from 2019-05-23 11-39-41](https://user-images.githubusercontent.com/45350418/58231310-1d10b280-7d50-11e9-88fd-1df57ce88a8c.png)

![screencapture-xpro-odl-local-8053-2019-05-23-11_46_12](https://user-images.githubusercontent.com/45350418/58231655-f2732980-7d50-11e9-924d-99461070bd3b.png)
![screencapture-xpro-odl-local-8053-2019-05-23-11_45_51](https://user-images.githubusercontent.com/45350418/58231668-f737dd80-7d50-11e9-8f63-4e870e006ad5.png)
